2022-05-metafacture-workshop/index.html

```diff
@@ -0,0 +1,939 @@
+				<p><span class="fragment"><em>Systemmigration</em>, z.B. nach Alma oder Folio</span>
+			</section>
+
+			<section>
+				<h2>Annahmen</h2>
+				<p><span class="fragment">Transformationen von Metadaten gehören zum täglichen Geschäft
+						wissenschaftlicher Bibliotheken.</span>
+				<p><span class="fragment">Es gibt viele unterschiedliche Methoden, die meist Programmierkenntnisse
+						voraussetzen.</span></p>
+				<p><span class="fragment">Datentransformationen werden meist im Zusammenspiel von Fachabteilungen &
+						IT umgesetzt, verbunden mit größerem Kommunikationsaufwand.</span></p>
+				<p><span class="fragment">Bereits existierende, von anderen entwickelte Transformationsprozesse können
+						nur bedingt entdeckt und nachgenutzt werden.</span></p>
+			</section>
+
+			<section>
+				<h2>Das heißt:</h2>
+				<p><span class="fragment">Es gibt großes Potential, eine immer wiederkehrende Arbeit zugänglicher,
+						kollaborativer und effizienter zu gestalten.</span>
+			</section>
+
+			<section>
+				<h2>Übergeordnete Ziele</h2>
+				<p><span class="fragment">Ermächtigung der Fachebene zur Konfiguration von Datentransformationen</span>
+				</p>
+				<p><span class="fragment">Förderung von Praktiken zum Teilen und Auffinden von
+						Transformationsprozessen</span></p>
+			</section>
+
+			<section>
+				<h1>2. Was ist Metafacture?</h1>
+			</section>
+
+			<section>
+				<h2>Was ist Metafacture?</h2>
+				<p><span class="fragment">[...] zur Verarbeitung von semi-strukturierten Daten mit dem Fokus auf
+						Bibliotheksdaten</span>
+				<p><span class="fragment">Ein vielseitiges Werkzeug: nutzbar als Kommandozeilentool, als Java/JVM
+						library, für Batch-Verarbeitung oder on-the-fly</span></p>
+				<p><span class="fragment">Ein offenes Framework: Wiederverwendung und Austausch (von einzelnen Modulen
+						und ganzen Workflows)</span></p>
+			</section>
+
+			<section>
+				<h2>Metafacture-Historie</h2>
+				<p><span class="fragment">2011: Start der Entwicklung durch DNB im Rahmen von Culturegraph; damals schon Austausch mit dem hbz</span></p>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-core/graphs/contributors">2013: Umzug auf GitHub, Open-Source-Projekt geworden</a></span></p>
+				<p><span class="fragment">2019: Mit der Zeit immer weniger DNB-Ressourcen für Metafacture, hbz wird
+						Maintainer</span></p>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-fix/graphs/contributors">2019: Start von Metafacture Fix</a></span></p>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-playground/graphs/contributors">2021: Start von Metafacture Playground</a></span></p>
+			</section>
+
+			<section>
+				<h2>Wie Metafacture funktioniert</h2>
+				<p><span class="fragment">Grundidee: Daten fließen durch mehrere Module: <br />&rarr; read &rarr; decode
+						&rarr; transform &rarr; encode &rarr; write &rarr;</span></p>
```

Comment:
```suggestion
				<p><span class="fragment">Grundidee: Daten fließen durch mehrere Module: <br />&rarr; read &rarr; decode
						&rarr; transform &rarr; encode &rarr; write &rarr;</span></p>
```

Die Abfolge fetten und auf der Folie absetzen.

Reply:
Eventuell auch die Abfolge der Module einfach auf die nächste Folie

Reply:
Eventuell Beispiele für Module geben und die auf den Zusatzseiten angeben.